MachUpskillingFY17/JabbR-Core
# Kevin/admin ban

**PR Changes:**
- Implemented CheckBannedCommand.cs
- Implemented BanCommand.cs
- Implemented UnBanCommand.cs
- Removed ForgotPasswordConfirmation [HttpGet] - Not necessary, redirects from Account/ForgotPassword
- Removed/Refactored unused AccountController functions (not referenced or used)
- Removed Account/Login Controller (Fake Jane info)

Closes #245, #213 , #247 

**Testing:**
_User running command MUST be an admin, update the 'IsAdmin' value manually in the DB_
1. Login with two separate browsers, one admin, one non-admin.
2. Test "/checkbanned" -> displays: no users banned.
3. Test with UserA: "/checkbanned UserB" -> displays: UserB is not banned
4. Test with UserA: "/ban UserB" -> displays: UserB has been banned, UserB should receive a notification that they have been banned by UserA and brought to the loginscreen.
5. Try logging in with UserB, they should be directed to a "BannedNotification" view.
6. Test with UserA: "/checkbanned" -> Should display list containing UserB
7. Test with UserA: "/checkbanned UserB" -> displays: UserB is banned.
8. Test with UserA: "/unban UserB" -> displays: UserB has been unbanned.
9. Test with UserA: "/checkbanned" -> Should display an empty list.
10. Test with UserA: "/checkbanned UserB" -> displays: UserB is not banned.
11. You should now be able to log back in with UserB.
12. Test with a third browser/user if they are in the same room as UserB when they are banned, they should see a notification that UserB has been banned.

Thanks, let me know if you have any questions.
